dev_guide/builds.adoc

```diff
@@ -317,16 +318,16 @@ JSON within the `*BuildConfig*`:
 The payload URL is returned as the GitHub Webhook URL by the `describe` command
 (see link:#describe-buildconfig[below]), and is structured as follows:
 
-****
-`http://_<openshift_api_host:port>_/osapi/v1/namespaces/_<namespace>_/buildconfigs/_<name>_/webhooks/_<secret>_/github`
-****
+----
+http://_<openshift_api_host:port>_/osapi/v1/namespaces/_<namespace>_/buildconfigs/_<name>_/webhooks/_<secret>_/github
```

Comment:
You can kill the underscores around the replaceables since they are not mark-up in a ---- block.  Same goes for another instance below.